src/rating/rating.jsx

```diff
@@ -419,6 +426,8 @@ export default class Rating extends Component {
                 className={ratingCls}
                 onKeyDown={this.onKeyDown}
                 tabIndex="0"
+                role="group"
+                aria-label={locale.description}
```

Comment:
- remove `aria-labelledby={id}` on line 365
- add `aria-hidden` to underlay div on line 435

Reply:
done